docusaurus-docs/conda-store/how-tos/install-standalone.md

```diff
@@ -0,0 +1,60 @@
+---
+sidebar_position: 1
+description: How to install conda-store locally in standalone mode
+---
+
+# Standalone (local) installation
+
+The standalone mode is an **experimental feature**, and the quickest way to start using conda-store.
```

Comment:
Is it still considered an experimental feature? Or is this also old language that should go?